packages/table/src/Row/InternalRowWithRT.tsx

```diff
@@ -86,8 +86,10 @@ const InternalRowWithRT = <T extends LGRowData>({
       <InternalRowBase
         className={cx(
           {
-            [grayZebraRowStyles[theme]]: isOddVSRow && shouldAlternateRowColor,
-            [zebraStyles[theme]]: !virtualRow && shouldAlternateRowColor,
+            [grayZebraRowStyles[theme]]:
+              isOddVSRow && shouldAlternateRowColor && !isSelected,
```

Comment:
nit: should we break `shouldAlternateRowColor && !isSelected` into a separate variable?

Reply:
Also, does this style depend on `!virtualRow` as well?

Reply:
`isOddVSRow` should imply `!!virtualRow`, so shouldn't be necessary!